website/docs/troubleshooting.md

```diff
@@ -4,18 +4,23 @@ sidebar_position: 4
 
 # Troubleshooting
 
-Here, find some answers to the question : I have downloaded Podman Desktop but I am unable to see any of my image or container
+This section of documentation provides way to troubleshoot problems with Podman Desktop.
 
-## Podman
+## Podman Issues
 
-### System Requirements
+### Getting Started
```

Comment:
```suggestion
### Unable to see any image or container after downloading Podman Desktop
```

Reply:
Are you sure that's really the case? 
Is this not something handled by pre-flight checks?